lib/gooddata/core/splunk_logger.rb

```diff
@@ -0,0 +1,138 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  class SplunkLogger < Logger
```

Comment:
Assumes too much for instance variable '@default_formatter' (InstanceVariableAssumption)